cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,80 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
+ url = '/settings';
+
+ editUsername (username) {
+  cy.getByDataCy('username-settings')
+    .type('{selectAll}' + username)
+ };
+
+ editBio (bio) {
+  cy.getByDataCy('bio-settings')
+    .type('{selectAll}' + bio)
+ };
+
+ editEmail (email) {
+  cy.getByDataCy('email-ettings')
+    .type('{selectAll}' + email)
+ };
+
+ editPassword (password) {
+  cy.getByDataCy('password-settings')
+    .type('{selectAll}' + password)
+ };
+
+ sumbitSettings () {
```

Comment:
Add proper naming to this method. If it includes any buttons clicking - the method name must reflect it.